tr/django_models/README.md

```diff
@@ -1,52 +1,54 @@
 # Django modelleri
 
-Şimdi blogumuzdaki bütün yazıları kaydedebileceğimiz bir şey oluşturmak istiyoruz. Ama bunu yapabilmek için önce `nesneler`den (objects) bahsetmemiz gerekiyor.
+Şimdi blogumuzdaki bütün yazıları kaydedebileceğimiz bir şey oluşturmak istiyoruz. Ama bunu yapabilmek için önce `nesneler` denen şeylerden bahsetmemiz gerekiyor.
 
 ## Nesneler
 
-Programlamada `Nesneye yönelik programlama` denen bir kavram bulunuyor. Buradaki ana fikir her şeyi sıkıcı bir düzende programlama komutları ile yazmak yerine şeyleri modelleyip birbirleri ile nasıl etkileşime geçeceklerini tanımlayabilmek.
+Programlamada `Nesneye yönelik programlama` denen bir kavram bulunuyor. Buradaki ana fikir her şeyi sıkıcı bir düzende programlama komutları ile yazmak yerine şeyleri modelleyip birbirleri ile nasıl etkileşime geçeceklerini tanımlayabileceğimiz.
 
 Peki bir nesne nedir? Özelliklerin ve hareketlerin bir bütünüdür. Kulağa garip geliyor olabilir ama bir örnekle açıklayacağız.
 
-Eğer bir kediyi modellemek istiyorsak `Kedi` nesnesini oluştururuz ve bu nesne şöyle özelliklere sahip olur:`renk`, `yaş`, `ruh hali` (örneğin; iyi, kötü, uykulu ;)) ve `sahibi` (ki bu da bir `İnsan` nesnesi olur ya da eğer sokak kedisi ise bu özellik boş olabilir).
+Eğer bir kediyi modellemek istiyorsak `Kedi` nesnesini oluştururuz ve bu nesne şöyle özelliklere sahip olur:`renk`, `yas`, `ruh_hali` (örneğin; iyi, kötü, uykulu ;)) ve `sahibi` (ki bu da bir `İnsan` nesnesi olur ya da eğer sokak kedisi ise bu özellik boş olabilir).
 
-`Kedi` bazı hareketlere sahiptir: `miyavla`, `tırmala` ya da `beslen` (bu durumda kediye biraz `Kedi Maması` vermemiz gerekir ki bu özelliklere sahip başka bir nesne olur. Özelliklere örnek olarak `tat` verilebilir).
-
-    Kedi
-     --------
-    renk
-    yaş
-    ruh_hali
-    sahibi
-    miyavla()
-    tırmala()
-    beslen(kedi_maması)
-
-    KediMaması
-     --------
-    tat
+`Kedi` bazı hareketlere sahiptir: `miyavla`, `tirmala` ya da `beslen` (bu durumda kediye biraz `KediMamasi` vermemiz gerekir ki o da kendine ait özellikleri olan başka bir nesne olur. Özelliklere örnek olarak `tat` verilebilir).
 
+```
+Kedi
+--------
+renk 
+yas 
+ruh_hali 
+sahibi 
+miyavla() 
+tirmala() 
+beslen(kedi_mamasi) 
+
+KediMamasi
+-------- 
+tat
+```
 
-Yani aslında ana fikir, gerçek nesneleri kod içinde özellikleriyle (`nesne özellikleri`) ve hareketleriyle (`metodlar`) tanımlamak.
+Yani aslında ana fikir, gerçek nesneleri kod içinde özellikleriyle (`nesne özellikleri`) ve hareketleriyle (`metodlar`) tanımlamak.).
```

Comment:
This is a bit CrowdIn's fault I guess. See the English sentence (original one). The last right parenthesis is not in the translatable region (it is added automatically). Turkish translation does not need it (not all languages have the same word order, obviously), so we cannot really remove that parenthesis in CrowdIn. I'll update the file on GitHub, but I suppose we need a better solution than this. (Also see German and Korean translators had the same issue. I'm going to raise the issue with translations@djangogirls, to see if they can solve it.)  

![y](https://cloud.githubusercontent.com/assets/1781335/13731133/422b410c-e91f-11e5-988f-2db556fc1ea0.png)